samples/bluetooth/central_nfc_pairing/README.rst

```diff
@@ -0,0 +1,122 @@
+.. _central_nfc_pairing:
+
+Bluetooth: Central NFC pairing
+##############################
+
+The Central NFC pairing sample demonstrates Bluetooth LE out-of-band pairing using an :ref:`st25r3911b_nfc_readme` and the NFC TNEP protocol.
+You can use it to test the touch-to-pair feature between Nordic Semiconductor's devices with :ref:`st25r3911b_nfc_readme` and an NFC Tag device with Bluetooth LE support.
+
+The sample shows the usage of NFC NDEF :ref:`nfc_ch` and :ref:`tnep_poller_readme` with the :ref:`Connection Handover service<nfc_tnep_ch_readme>`.
+It provides minimal Bluetooth functionality in the Central role, and on GATT level it implements only the Device Information Service.
+
+The sample supports pairing in one of the following modes:
+
+* LE Secure Connections Just Works pairing
+* LE Secure Connections OOB pairing
+* Legacy OOB pairing
+* Legacy Just Works pairing
+* Negotiated Handover (TNEP protocol) - two-way OOB pairing
+
+Overview
+********
+
+When the sample starts, it initializes and starts the :ref:`st25r3911b_nfc_readme`, and waits for a Type 2 Tag or a Type 4 Tag to appear in the field.
+When the tag is detected, the sample identifies its type, then reads it, and parses its content.
+After successful parsing, the tag content is printed on the serial port.
+The sample does not start scanning immediately, but only when the NFC tag is read by an NFC Poller, or when the exchange of data over the TNEP Connection Handover Service is finished.
+
+NFC data exchange
+*****************
+
+This sample can read BLE OOB data from the Static NFC Tag or exchange the OOB data with an NFC Tag device with TNEP support.
+
+Static Tag
+==========
+A Static Handover Tag contains a Handover Select Message with carrier information NDEF records or a single Carrier Configuration Record.
+
+.. figure:: /images/nfc_static_connection_handover.svg
+   :scale: 100 %
+   :alt: Static Handover
+
+Negotiated Handover
+===================
+
+In Negotiated Handover, two NFC Forum Devices identify a matching set of alternative carriers by exchanging lists of potential alternative carriers.
+In case of the OOB data for BLE the NFC Devices can exchange OOB data 2-ways.
+Handover messages are exchanged using the TNEP Single response communication mode.
+The NFC Poler Device can be the Handover Requester or Handover Selector Device, the role is taken based on the NFC Tag first NDEF message.
+
+.. figure:: /images/nfc_negotiated_connection_handover.svg
+   :scale: 100 %
+   :alt: Negotiated Handover
+
+Requirements
+************
+
+* One of the following Nordic development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340pdk_nrf5340_cpuapp, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833
```

Comment:
Is nrf5340pdk_nrf5340_cpuapp the correct board target?

Reply:
It works on nrf5340pdk_nrf5340_cpuapp with Zephyr controller